actions.yaml

```diff
@@ -0,0 +1,12 @@
+authorize-charm:
+  description: >-
+    Authorizes the charm to be able to interact with Vault to manage its
+    operations. A token is required for Vault to use to create the app role and
+    the policy the charm will use to interact with Vault.
+  params:
+    token:
+      type: string
+      description: >-
+        A token for Vault that can create new policies. Used to create the app
+        role and policy for the charm.
```

Comment:
But, do we actually need a root token?

I guess I'm fine if we want to simplify it, but I tried to avoid that because I didn't think we actually specifically needed a root token, and I didn't want to artificially restrict it.

Reply:
My thinking was it might be confusing for the user. When they run init, they get an `unseal_key` and a `root_token`. This to me kinda implies that they need to make a unique token or policy just for this purpose.

Reply:
Documentation updated in https://github.com/canonical/vault-operator/pull/62/commits/3061b56129d9f395a7f9c3f4ed05694a2b7c160a